Description: Use local logo image to fix privacy-breach-logo lintian error.
Author: Bas Couwenberg <sebastic@xs4all.nl>
Forwarded: not-needed

--- a/docs/obsolete/fan_utils.html
+++ b/docs/obsolete/fan_utils.html
@@ -2,7 +2,7 @@
 
 <head>
 <!-- InstanceBeginEditable name="Title" -->
-<TITLE>Introduction to FAN Language and Utilities </TITLE>    <link rel="shortcut icon" href="https://www.unidata.ucar.edu/favicon.ico" type="image/x-icon" />
+<TITLE>Introduction to FAN Language and Utilities </TITLE>
 
 </head>
 
